src/containers/FormikForm/TitleField.tsx

```diff
@@ -18,27 +20,48 @@ interface Props {
   maxLength?: number;
   name?: string;
   type?: string;
+  isTitle?: boolean;
```

Comment:
Likte ikke dette navnet helt, at en komponent som heter `TitleField` har en prop som heter `isTitle` gir ikke helt mening for meg🤔